features/step_definitions/setup_steps.rb

```diff
@@ -279,7 +279,7 @@ def delete_all_draft_briefs (user_id)
   end
 end
 
-Given /^I have a '(.*)' brief$/ do |brief_state|
+Given /^I have a '(.*)' (?:brief|opportunity|set of requirements)$/ do |brief_state|
```

Comment:
It would be nice to purge `brief` and `briefs` altogether, but not necessarily for this PR.